cmd/ftl/main.go

```diff
@@ -82,6 +82,10 @@ func main() {
 	logger := log.Configure(os.Stderr, cli.LogConfig)
 	ctx = log.ContextWithLogger(ctx, logger)
 
+	err := projectconfig.CreateFilesIfNonexistent(ctx, cli.ConfigFlag)
+	if err != nil {
+		kctx.Fatalf(err.Error())
```

Comment:
You can use `kctx.FatalIfErrorf(err)` to skip the `if` statement

Reply:
nice, thanks!